libbeat/outputs/elasticsearch/client.go

```diff
@@ -472,7 +477,7 @@ func itemStatus(reader *jsonReader) (int, []byte, error) {
 	}
 
 	// parse actual item response code and error message
-	status, msg, err := itemStatusInner(reader)
```

Comment:
It was probably an oversight that it didn't check the `err`. Instead of ignoring it, an error check should be added.